ansible/testbed-cli.sh

```diff
@@ -644,6 +712,8 @@ case "${subcmd}" in
                ;;
   deploy-mg)   deploy_minigraph $@
                ;;
+  activate-wan-sonic-device)   activate_wan_sonic_config $@
+               ;;
```

Comment:
Please update the help text of this command line tool to include the newly added sub-commands.